### PR TITLE
[alarmdecoder] Workaround for thing actions bug

### PR DESCRIPTION
**Pull request by bobadair**
This update applies a workaround for issue openhab/openhab-core#1265 as suggested in #8116.

### CODE REVIEW

File: bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/actions/BridgeActions.java [hunk 1]
```diff
@@ -68,10 +72,34 @@ public void reboot() {
...
+            } else {
+                return (IBridgeActions) Proxy.newProxyInstance(IBridgeActions.class.getClassLoader(),
+                        new Class[] { IBridgeActions.class }, (Object proxy, Method method, Object[] args) -> {
+                            Method m = actions.getClass().getDeclaredMethod(method.getName(),
+                                    method.getParameterTypes());
+                            return m.invoke(actions, args);
+                        });
+            }
         }
+        throw new IllegalArgumentException("Actions is not an instance of VolvoOnCallActions");
```
Comment (lolodomo) [inline bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/actions/BridgeActions.java:103]: Please fix the message.

Comment (bobadair) [reply bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/actions/BridgeActions.java]: Thanks. I missed that.

**Comment by cpmeister** [comment]
@bobadair Can you fix the DCO check?

**Comment by bobadair** [comment]
Not sure why the DCO check is failing. All of the commits are signed off.

**Comment by cpmeister** [comment]
Manually checked sign-off

**Pull request merged**